packages/react-sdk/src/css/presets.ts

```diff
@@ -18,3 +18,21 @@ export const borders: EmbedTemplateStyleDecl[] = [
     value: { type: "unit", value: 1, unit: "px" },
   },
 ];
+
+export const linkColors: EmbedTemplateStyleDecl[] = [
```

Comment:
we should really rename EmbedTemplate because for example here it's not an embed any more

Reply:
We reuse same simplified type. It's just resolved differently. I can alias it on top of module to make it less confusing.